roles/validate/files/defaults.yml

```diff
@@ -46,6 +48,12 @@ factory_defaults:
         bridge_priority: 0
       netflow:
         enable: false
+      ptp:
+        ptp_enable: false
+        ptp_domain_id: 0
+        ptp_lb_id: 0
+      enable_nxapi: true
+      enable_nxapi_http: true
```

Comment:
@mikewiebe can you (we) review these with respect to d2d that @rafmuller was looking at for device specific; i.e. if a device also has these like we talked maybe we enable fabric wide for dtc vs defining in model as such...thoughts???